packages/charting/src/components/CommonComponents/CartesianChart.types.ts

```diff
@@ -157,6 +157,12 @@ export interface ICartesianChartProps {
 
   /**
    * Margins for the chart
+   * @default top: 20
+   * @default bottom: 35
+   * @default left: 40
+   * @default right: 20
```

Comment:
You should only provide a single value for `@default`, and use backtick syntax to wrap the object to prevent some parsers from complaining.
```suggestion
   * @default `{ top: 20, bottom: 35, left: 40, right: 20 }`
```

Reply:
Done, 
Thank you!!!